custom-routes-vis/src/main/webapp/script.js

```diff
@@ -12,9 +12,105 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Creates the world map. */
+// Array holding origin and destination markers.
+let markers = []
```

Comment:
Could you rename this to something more specific? Like include 
what these markers are for or something like that?